### PR TITLE
fix (horizontal scroll) in goTo 

### LINKED ISSUE #6914: [Bug Report] goTo vertical scroll does not work on iOS
### Versions and Environment
**Vuetify:** 1.5.9
**Vue:** 2.6.10
**Browsers:** Safari
**OS:** Mac OS 10.14.3

### Steps to reproduce
On iOs device (any browser) or Safari (mac os) click any of the menus (one, two, three).

### Expected Behavior
It should scroll, after menu click, vertically to the corresponding IDs

### Actual Behavior
It does not scroll on, menu click, on iOs device (any browser) or Safari (mac os). 

After scrolling the window down a bit then it works fine.

### Reproduction Link
<a href="https://ww9nq0kq48.codesandbox.io" target="_blank">https://ww9nq0kq48.codesandbox.io</a>

### Other comments
Found problem in goTo function and sending trough pull request soon.

### PR DESCRIPTION
**Pull request by raybog**
## Description
Horizontal scroll fix for goTo

## Motivation and Context
fixes #6914

## How Has This Been Tested?
Tested on latest Safari/Chrome/FF both iOS and macOS

## Markup:

<details>

```vue
// n/a
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x ] The PR title is no longer than 64 characters.
- [ x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/Vuetify/goTo/index.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default function goTo (_target: GoToTarget, _settings: Partial<GoToSettin
 
     container.scrollTop = Math.floor(startLocation + (targetLocation - startLocation) * ease(progress))
 
-    if (progress === 1 || container.clientHeight + container.scrollTop === container.scrollHeight) {
+    if (progress === 1 || window.innerHeight + container.scrollTop === container.scrollHeight) {
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/Vuetify/goTo/index.ts]: does it still work if you use the container?

Comment (raybog) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: With container (`container.clientHeight`) works fine in Chrome and FF on macOS but not iOS.
Safari is impacted on both macOS and iOS.

With proposed change (`window.innerHeight`) works fine on all 3 browsers in macOs and iOS.

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: I mean `goto({ container: ..., ...})`

Comment (raybog) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: ...sorry
when you pass container via goTo then inside the`container.scrollTop` is always 0 (it is read only?)

Comment (raybog) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: Just had another look and perhaps even better solution to this problem (covering both scroll down and up) will be to change following:

`if (progress === 1 || container.clientHeight + container.scrollTop === container.scrollHeight)`
into
`if (progress === 1 )` - tested and works well.

However not sure what is the purpose of `container.clientHeight + container.scrollTop === container.scrollHeight` ?

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: It stops the animation when scroll goes beyond the maximum value (for example you have small container but you want to do `goTo(aBigNmber)`)

Comment (raybog) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: oh then perhaps this maximum value should be checked and adjusted before `requestAnimationFrame` is called?

Comment (johnleider) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: Yea let's do that.

Comment (raybog) [reply packages/vuetify/src/components/Vuetify/goTo/index.ts]: do you want me to create another pr where maximum value should be checked and adjusted before `requestAnimationFrame` is called?

**Comment by johnleider** [comment]
This Pull Request is being closed due to inactivity.

Thank you for your contribution and interest in improving Vuetify!

**Pull request closed without merging**